### PR TITLE
feature: Add unified global exception handling using AlertDto

### PR DESCRIPTION
**Pull request by KinTrae**
Developer tests and documentation: 
[[Testy dev] EPIC - API v3 - create AlertDto and global error handling.docx](https://github.com/user-attachments/files/18223222/Testy.dev.EPIC.-.API.v3.-.create.AlertDto.and.global.error.handling.docx)
Link: https://pejot.sharepoint.com/:w:/s/MeowHub/ERF5iCgXBmJLvA5D_0Kz7cIBIp3kvYWeeeYxIeV3DbTviA?e=FaoUHG

Task: 8696zwhj1

Preview of the attached document: 
image

### CODE REVIEW

**Comment by JimTheCat** [comment]
Task linked: [CU-8696y76jy EPIC - API v3](https://app.clickup.com/t/8696y76jy)

File: backend/src/main/java/meowhub/backend/shared/constants/AlertConstants.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+    //message
+    public static final String USER_WITH_LOGIN_NOT_FOUND = "User with login '%s' not found";
+    public static final String RESOURCE_NOT_FOUND = "%s not found for %s = '%s'";
+    public static final String UNKNOWN_ERROR = "Unknown error";
+    public static final String BAD_CREDENTIALS = "Bad credentials";
+    public static final String NOT_UNIQUE_OBJECT = "%s:'%s' is not unique";
+    public static final String ILLEGAL_ARGUMENT = "%s cannot be equal %s";
+
+    private AlertConstants() {
```
Comment (JimTheCat) [inline backend/src/main/java/meowhub/backend/shared/constants/AlertConstants.java:18]: Annotation instead of constructor like @NoArgsConstructor?

**Review by JimTheCat — CHANGES_REQUESTED** [review]
Please take a look on this one comment :-)

File: backend/src/main/java/meowhub/backend/shared/constants/AlertConstants.java (hunk 1 above)
Comment (KinTrae) [reply backend/src/main/java/meowhub/backend/shared/constants/AlertConstants.java:18]: My bad. Must have slipped my mind. I'll use the annotation

**Review by JimTheCat — APPROVED** [review]
Sexy

**Pull request merged**